terraform/github/repositories/data.tf

```diff
@@ -5,4 +5,9 @@ data "github_team" "operations_engineering" {
 data "github_team" "operations_engineering_test" {
   provider = github.ministryofjustice-test
```

Comment:
Do we need 2 test teams?

Reply:
Well one I used when I created the test repo so then I created a new test team to see what would happen when trying to import to that. But the teams themselves are all click-ops'd. We probably don't need so many test repos. How about we do that in a tidy up ticket?